controllers/v1/forms.js

```diff
@@ -18,6 +21,94 @@
    static get name() {
      return "forms";
    }
- 
- };
+
+       /**
+   * @api {post} /kendra/api/v1/forms/list
```

Comment:
@aman-tunerlabs - Please make it a details API and take form name as a parameter in URL